website/content/accessibility/deaf-and-hard-of-hearing/conference-best-practices.md

```diff
@@ -0,0 +1,141 @@
+
+## Website, registration and CFP form, and program information
+
+### Event website
+
+On the registration and call for papers (CFP) pages, state that accommodations will be made for speakers and attendees with disabilities. Clearly and prominently stating that will make us feel welcome and motivate us to participate. Don't bury that important information in a registration form we might never see. Here's some suggested copy:
+
+> Interested in attending or speaking but require special accommodations? Please inform us through the registration or CFP form about your specific needs, and we will make every effort to provide the necessary support. For accommodation-related questions, please reach out to [accommodations@yourevent.com](mailto:accommodations@yourevent.com).
+
+Consider creating a dedicated email address (e.g., accommodations@) to signal you take this seriously.  
+
+### Registration and CFP form
+
+Include a field asking if accommodations are needed (yes/no) and a free-form text box to specify. Everyone's needs differ; it's hard to discern from a fixed form/dropdown.
+
+### Program Information
+
+On the program page, include session accessibility information, such as whether captioning will be provided and in which language. Also, mention that interpreters will be available upon request if requested in advance, and specify which sign language (e.g., American Sign Language, International Sign Language, etc.).  
+
+## Pre-conference best practices 
+
+### Reach out to attendees/speakers before the event
+
+* Set up notifications or monitor registration and CFP forms for accommodation needs and ask for clarification if needed.
+* Connect attendees with sign language interpreters to ensure they understand communication needs and style.
+* For dedicated interpreters (see the [sign language interpreters](#sign-language-interpreters) section), provide both sides with contact information to communicate directly during the event should they run late or if a meeting place changes.  
+
+### Connect us with our peers
+
+Provide a way for deaf and hard of hearing attendees and speakers or people with other disabilities to connect for peer support and potentially pool resources (e.g., an event Slack channel). 
+
+## Day of best practices
+
+Needs vary. Interpreters, captions, or both may be necessary for different individuals.
+
+### Reserved seating at the front
+
+Have some reserved seating at the front for deaf and hard of hearing attendees. Many rely on lip-reading, so you want to be sure they can see the speakers well. And please ask speakers not to wear masks while presenting and provide social distancing opportunities instead. 
```

Comment:
```suggestion
Have some reserved seating at the front for deaf and hard of hearing attendees. Many rely on lip-reading, so you want to be sure they can see the speakers well. If speakers are wearing masks, it inhibits many deaf/hoh attendees from reading lips. Opt to provide speakers with clear masks if they prefer to wear a mask when speaking, or provide social distancing opportunities instead so they may speak maskless. 
```
I know the research on clear masks is limited in this space, however conference organizers should be mindful that some speakers may prefer to mask when speaking so balancing both needs is important and i _think_ clear masks are a method to do that. Please let me know if that is not accurate.